maple2latex/src/translator.py

```diff
@@ -77,11 +201,11 @@ def replace_strings(string, keys):
     return string
 
 
-def parse_brackets(exp):
+def parse_brackets(string):
     # type: ((str, list)) -> list
     """Obtains the contents from data encapsulated in square brackets."""
 
-    exp = exp[1:-1].split(",")
+    exp = string[1:-1].split(",")
     for i, e in enumerate(exp):
```

Comment:
Instead of replacing from this loop on every element of the split string, you can just do the replacement on the original string before splitting.